Governance process been actualized. (#82736)
pytorch/pytorch: docs/source/community/governance.rst

The Process for Nomination
~~~~~~~~~~~~~~~~~~~~~~~~~~

> * Each module will have it's own process, please conact module maintainers

Reviewer: ```suggestion
* Each module has its own process. Please contact module maintainers for more information.
```

>   however if there is no one, you can file a request to the core maintainers

Reviewer: ```suggestion
  However, if there is no process identified, you can file a request to the core maintainers
```

>   via [this form](https://forms.gle/xNeu1byGMZVHcA2q7). Core maintainers are

Reviewer: ```suggestion
  by submitting [this form](https://forms.gle/xNeu1byGMZVHcA2q7). Core maintainers are
```

>   meeting each 3 months.

Reviewer: ```suggestion
  meeting every three months.
```
Should we say once a quarter?

> * If you using the core maintainers route the information in your reqest 

Reviewer: ```suggestion
* If you are submitting a request to the core maintainers, the information in your request 
```

>   should include the following items:

Reviewer: ```suggestion
  must include the following items:
```
